delocate/tools.py

```diff
@@ -290,7 +290,9 @@ def set_install_id(filename, install_id):
 RPATH_RE = re.compile(r"path (.*) \(offset \d+\)")
 
 def get_rpaths(filename):
-    """ Return a tuple of rpaths from the library `filename`
+    """ Return a tuple of rpaths from the library `filename`, with an
```

Comment:
This extra part of the comment doesn't apply to this function, does it?

Reply:
Quite right.